.github/workflows/markdown-links.yml

```diff
@@ -0,0 +1,42 @@
+name: Check Markdown links
+
+on: [push, pull_request]
+
+env:
+  PYVER: "3.8"
+
+jobs:
+  markdown-link-check-push:
+    if: github.event_name == 'push' && github.repository_owner != 'conan-io' # We do not want to see red in CCI
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Get changed files
+      uses: ./.github/actions/pr_changed_files
+      id: changed_files
+      with:
+        files: |
+          docs/**
+    - uses: gaurav-nelson/github-action-markdown-link-check@v1
+      if: steps.changed_files.outputs.any_changed == 'true'
+      with:
+        config-file: .github/workflows/mlc_config.json
+
+  markdown-link-check-pr:
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Get changed files
+      uses: ./.github/actions/pr_changed_files
+      id: changed_files
+      with:
+        files: |
+          docs/**
```

Comment:
The pattern expected for the `files` parameters of this action should be "bash" compatible, because the same variable is used as [bash expansion](https://github.com/conan-io/conan-center-index/blob/master/.github/workflows/linter-yaml.yml#L59) and as [action parameter](https://github.com/conan-io/conan-center-index/blob/master/.github/workflows/linter-yaml.yml#L105) in other workflows.
I don't think `docs/**` will work for this reason. You could try with
```
docs/*.md
docs/*/*.md
```
But this would have to be completed if we add another layer of subfolder in the docs folder :/

Reply:
Hi @ericLemanissier and thank you for the feedback.
Regarding this line, I just copied the pattern from https://github.com/conan-io/conan-center-index/blob/2c6c5715f2e18f25f6bce8be24eacae83924d943/.github/workflows/linter-yaml.yml#L24, isn't that valid in that case either?

Reply:
After checking implementation of the action, double wildcard are actually interpreted as single wildcard, meaning "any file is modified in this folder". It ignores all modifications in a sub-folder.

Reply:
thanks for checking. So it would be difficult to add something that tracks all the changes under `docs` folder. Could you help me with the filtering for the best way to add this? A code suggestion in a review would be awesome 😄

Reply:
What I said earlier was wrong: it's a plain limitation of the github action, it's not linked in any way with bash expansion

Reply:
Re-rectification: bash (and shell) expansion actually treats `**` as `*`. I briefly thought otherwise because I decided to test it, but I did not realize I was using fish shell, which supports `**`, contrary to bash and shell...